frontend/src/components/createUnionComponent.vue

```diff
@@ -208,6 +208,13 @@ img {
   }
 }
 
+.animation {
```

Comment:
Dit heeft nu dezelfde naam als die van theme. Weet je zeker dat je die in theme niet wat meer generic kan noemen. animation-small bijvoorbeeld?

Dan zou je de an animation-medium en animation-big combo er ook ooit bij kunnen zetten.